components/ShakeApi.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from "react";
+import { View, Text, TouchableOpacity, Image, StyleSheet } from "react-native";
+import { Accelerometer } from "expo-sensors";
+
+const styles = StyleSheet.create({
```

Comment:
Nice job with the stylesheet styled components! I like the clean design which really lets the foxes shine!